robottelo/datafactory.py

```diff
@@ -32,32 +33,37 @@ def func_wrapper(*args, **kwargs):
 
 
 @datacheck
-def generate_strings_list(length=None, remove_str=None, bug_id=None):
+def generate_strings_list(length=None, exclude_types=None, bug_id=None,
```

Comment:
@elyezer did not know about it, I'll change it and pass a **tuple**, Also we need to change the Docstring which says **length** is int type.